rero_ils/modules/documents/views.py

```diff
@@ -207,6 +207,12 @@ def can_request(item):
     return False
 
 
+@blueprint.app_template_filter()
+def get_note(item, note_type):
+    """Get the public note corresponding to an item."""
```

Comment:
`Get the public note corresponding to an item`
As it is coded now,  it is get a note by its type